sweep/sweeper.go

```diff
@@ -886,12 +878,6 @@ func (s *UtxoSweeper) getInputLists(
 	// sweeper to avoid this.
 	var newInputs, retryInputs []txInput
 	for _, input := range cluster.inputs {
-		// Skip inputs that have a minimum publish height that is not
-		// yet reached.
-		if input.minPublishHeight > s.currentHeight {
-			continue
-		}
```

Comment:
are we sure we want to remove this?

Reply:
yeah because we are not managing maturity of inputs in the sweeper - the inputs are only offered by the resolvers when they are mature.